api/v1alpha1/bmc_types.go

```diff
@@ -160,8 +160,8 @@ const (
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Cluster
 // +kubebuilder:printcolumn:name="MACAddress",type=string,JSONPath=`.status.macAddress`
-// +kubebuilder:printcolumn:name="MACAddress",type=string,JSONPath=`.status.ip`
-// +kubebuilder:printcolumn:name="Model",type=string,JSONPath=`.status.model`
+// +kubebuilder:printcolumn:name="IP",type=string,JSONPath=`.status.ip`
+// +kubebuilder:printcolumn:name="Model",type=string,JSONPath=`.status.model`,priority=100
```

Comment:
Do we want to hide the model behind `-o wide`? I would say lets print it out by default.

Reply:
sounds good, by default makes sense